openapi.yaml

```diff
@@ -1607,7 +1607,6 @@ components:
       type: object
       required:
         - addressType
-        - addressLine1
```

Comment:
I ran into this error now `"Error: ORA-01008: not all variables bound"`

And this is my data packet:

```json
{
  "data": {
    "type": "addresses",
    "attributes": {
      "addressType": {
        "code": "XX"
      },
      "houseNumber": "123",
      "city": "Corvallis",
      "stateCode": "OR",
      "postalCode": "97331",
      "nationCode": "FPL"
    }
  }
}
```